Dockerfile

```diff
@@ -28,7 +28,8 @@ RUN export MYSQL_DRIVER_VERSION=5.1.38 && \
       gzip \
       curl \
       su-exec \
-      wget &&  \
+      wget \
+      zip &&  \
```

Comment:
Zip dependency have to be added to the cleanup routine "apk del" to remove dependency after use